src/test/java/com/mgazanayi/bug/HttpServerTest.java

```diff
@@ -34,13 +34,29 @@ public void close(TestContext context) {
    }
 
    @Test
-   public void should_call_vertx_http_server(TestContext testContext) {
+   public void should_call_vertx_http_server_1(TestContext testContext) {
```

Comment:
Good catch! 
I suggest to inline the content and name the different tests alphabetically to show that it is always the first to run which fails only.